R/db-odbc-mssql.R

```diff
@@ -68,7 +68,28 @@
       as.double     = sql_cast("NUMERIC"),
       as.character  = sql_cast("VARCHAR(MAX)"),
       log           = sql_prefix("LOG"),
+      nchar         = sql_prefix("LEN"),
+      atan2         = sql_prefix("ATN2"),
+      substr        = sql_prefix("SUBSTRING"),
```

Comment:
Good point,`SUBSTRING` uses x, start length, while `substr()` uses x, start, stop, I'll fix it.  Thank you